modules/ROOT/pages/monitoring/metrics/reference.adoc

```diff
@@ -437,6 +437,17 @@ label:deprecated[Deprecated in 5.15]
 |<prefix>.cluster.discovery.restart.failed_count|Discovery restart failed count. (gauge)
 |===
 
+[[discovery-metrics]]
```

Comment:
```suggestion
[[discovery-metrics-v2]]
```

Reply:
This will be a new metric class?

Reply:
our test shows that we have to have the metric class - Lighthouse metrics. I suppose we should rename 'Discover metrics v2' into 'Lighthouse metrics'